app/src/main/java/ar/com/wolox/android/example/ui/example/presenter/NewsPresenter.kt

```diff
@@ -24,4 +37,73 @@ class NewsPresenter @Inject constructor() : BasePresenter<NewsView>(){
     }
 
     fun getNewsRowsCount() : Int = items.size
+
+    fun getNews(fromPullRefresh : Boolean) = launch{
+        // Si es por pull refresh consulto page=1 y no muestro progressbar
+        var currentpage=page
+        if(fromPullRefresh)
+            currentpage=1
+        else
+            view?.showLoading(View.VISIBLE)
```

Comment:
We usually use curly brackets, even when the if/else statement has only one line